src/main/java/org/opensearch/cluster/etcd/ETCDHeartbeat.java

```diff
@@ -171,6 +173,7 @@ void publishHeartbeat() {
             heartbeatData.put("nodeId", nodeId);
             heartbeatData.put("ephemeralId", ephemeralId);
             heartbeatData.put("address", address);
+            heartbeatData.put("transportPort", transportPort);
```

Comment:
We need the name to match the one used here: https://github.com/opensearch-project/cluster-etcd/blob/b1ca31f50380738a11c9d73ce8270850c15a4317/src/main/java/org/opensearch/cluster/etcd/ETCDStateDeserializer.java#L633

Actually, let's create constants for all of these (in `ETCDHeartbeat`) and then we can reference them from `ETCDStateDeserializer`. That way we also have a compile-time guarantee that they're consistent.